### DISCUSSION TITLE
Huawei HarmonyOS Next SDK?

### DISCUSSION THREAD

**Discussion by RyzeKit** [logto-io/logto#5902] [Ideas]
Hi, will there be an SDK for Huawei's HarmonyOS Next when it comes out?

Anyways, keep up the good work! 💯 👍

**Comment by darcyYe** [comment]
Thanks for bringing this up for discussion, we haven’t seen much demand yet about supporting HarmonyOS SDK. When we see more user demand we will investigate the implementation and delivery of HarmonyOS SDK, but it's not on our short term roadmap.
As an open-source project, Logto always welcome contributions from the community for features that people wants!
